physics/lsm_ruc.F90

```diff
@@ -1060,81 +1115,50 @@ subroutine lsm_ruc_run                                            & ! inputs
           sneqv_lnd(i,j)=0
         endif
 
-        if(debug_print) then
-          if(me==0 ) then
-            write (0,*)'before LSMRUC for land'
-            write (0,*)'sfcems(i,j) =',i,j,sfcems_lnd(i,j)
-            write (0,*)'chklowq(i,j) =',i,j,chklowq(i,j)
-            write (0,*)'chs(i,j) =',i,j,chs_lnd(i,j)
-            write (0,*)'flqc(i,j) =',i,j,flqc_lnd(i,j)
-            write (0,*)'flhc(i,j) =',i,j,flhc_lnd(i,j)
-            write (0,*)'wet(i,j) =',i,j,wet(i,j)
-            write (0,*)'cmc(i,j) =',i,j,cmc(i,j)
-            write (0,*)'shdfac(i,j) =',i,j,shdfac(i,j)
-            write (0,*)'alb(i,j) =',i,j,alb_lnd(i,j)
-            write (0,*)'znt(i,j) =',i,j,znt_lnd(i,j)
-            write (0,*)'z0(i,j) =',i,j,z0_lnd(i,j)
-            write (0,*)'snoalb1d(i,j) =',i,j,snoalb1d_lnd(i,j)
-            write (0,*)'landusef(i,:,j) =',i,j,landusef(i,:,j)
-            write (0,*)'soilctop(i,:,j) =',i,j,soilctop(i,:,j)
-            write (0,*)'nlcat=',nlcat
-            write (0,*)'nscat=',nscat
-            write (0,*)'qsfc(i,j) =',i,j,qsfc_lnd(i,j)
-            write (0,*)'qvg(i,j) =',i,j,qvg_lnd(i,j)
-            write (0,*)'qsg(i,j) =',i,j,qsg_lnd(i,j)
-            write (0,*)'qcg(i,j) =',i,j,qcg_lnd(i,j)
-            write (0,*)'dew(i,j) =',i,j,dew_lnd(i,j)
-            write (0,*)'soilt(i,j) =',i,j,soilt_lnd(i,j)
-            write (0,*)'tskin(i) =',i,j,tskin_lnd(i)
-            write (0,*)'soilt1(i,j) =',i,j,soilt1_lnd(i,j)
-            write (0,*)'tsnav(i,j) =',i,j,tsnav_lnd(i,j)
-            write (0,*)'tbot(i,j) =',i,j,tbot(i,j)
-            write (0,*)'vtype(i,j) =',i,j,vtype_lnd(i,j)
-            write (0,*)'stype(i,j) =',i,j,stype_lnd(i,j)
-            write (0,*)'xland(i,j) =',i,j,xland(i,j)
-            write (0,*)'xice(i,j) =',i,j,xice(i,j)
-            write (0,*)'iswater=',iswater
-            write (0,*)'isice=',isice
-            write (0,*)'xice_threshold=',xice_threshold
-            write (0,*)'con_cp=',con_cp
-            write (0,*)'con_rv=',con_rv
-            write (0,*)'con_rd=',con_rd
-            write (0,*)'con_g=',con_g
-            write (0,*)'con_pi=',con_pi
-            write (0,*)'con_hvap=',con_hvap
-            write (0,*)'stbolt=',stbolt
-            write (0,*)'smsoil(i,:,j)=',i,j,smsoil(i,:,j)
-            write (0,*)'slsoil(i,:,j)=',i,j,slsoil(i,:,j)
-            write (0,*)'stsoil(i,:,j)=',i,j,stsoil(i,:,j)
-            write (0,*)'smfrsoil(i,:,j)=',i,j,smfrsoil(i,:,j)
-            write (0,*)'keepfrsoil(i,:,j)=',i,j,keepfrsoil(i,:,j)
-            write (0,*)'acrunoff(i,j) =',i,j,acrunoff(i,j)
-            write (0,*)'acsn(i,j) =',i,j,acsn(i,j)
-            write (0,*)'shdmin1d(i,j) =',i,j,shdmin1d(i,j)
-            write (0,*)'shdmax1d(i,j) =',i,j,shdmax1d(i,j)
-            write (0,*)'rdlai2d =',rdlai2d
-          endif
+        !if (debug_print) then
```

Comment:
Why was `if (debug_print) then` commented out? I doubt everyone wants to see this output for the first 10 timesteps every time they run, right?

Reply:
Comment is removed.